website/project/model.py

```diff
@@ -662,6 +660,17 @@ def __repr__(self):
                 'with _id {self._id!r}>').format(self=self)
 
     @property
+    def is_archiving(self):
+        return self.archiving or False
```

Comment:
Any reason for wrapping this in a property?
`or False` should not be here `archiving` will default to `False` and should never change type from `bool`